configuration/openmrs/apps/clinical/medication.json

```diff
@@ -0,0 +1,172 @@
+{
+  "commonConfig": {},
```

Comment:
What exactly differs in this file from Haiti one?
Isn't there a way to get things aligned? If not, then that's fine.

Reply:
Deleted to inherit directly from haiti